src/main/kotlin/calculator/StringAddCalculator.kt

```diff
@@ -11,12 +11,14 @@ class StringAddCalculator {
         println("계산 결과 : $sumResult")
     }
 
-    fun add(userInput: String? = ""): Int {
-        val tokenIterator = InputTokenizer(userInput).getTokensIterator()
+    fun add(userInput: String?): Int {
+        val tokenizer = InputTokenizer(userInput ?: "0")
```

Comment:
```suggestion
        if(userInput) {
               return 0;
        }
```

으로 빠른 반환을 해주면 더 좋지 않을까하네요~